src/components/MoviesList/component.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import MovieItem from 'Components/MovieItem'
+import { Col } from 'antd'
+
+const MoviesList = ({ moviesIDs, moviesData }) => (
+  moviesIDs && moviesIDs.map((item) => {
```

Comment:
you can avoid of extra check of presence of `moviesIDs` by using `map` fron `lodash`